jobs/service-fabrik-report/templates/config/settings.yml.erb

```diff
@@ -59,6 +59,16 @@ production:
     <% end.else do %>
     protocol: 'http'
     <% end %>
+
```

Comment:
why does report need this config?

Reply:
report process does not need this setting per se. But currently `deploymentHookClient ` exports client instance which is loaded by `DirectorManager`. And `DirectorManager` is loaded by `index.js` of lib. So it becomes mandatory for processes using broker module to have deployment_hooks settings :(
So in current situation we have to add almost all the configs from broker to report as well as scheduler.
One more reason to modularise our source code.

Reply:
in current setup as well you can avoid it by exporting `DeploymentHookClient` class and not its instance and DirectorManager lazily creating an instance of it when required. That should also address the above isssue?

Reply:
Done!